zuul.d/jobs.yaml

```
@@ -863,88 +951,123 @@
       - huaweicloud_credentials
 
 - job:
-    name: docker-machine-functional-devstack-mitaka
+    name: docker-machine-functional-devstack
     parent: init-test
     description: |
-      Test image building functionality of Docker machine of v0.16.1 version against
-      devstack of Mitaka.
+      Test image building functionality of master Docker machine against master devstack.
```

```suggestion
      Test image building functionality of master Docker machine against master OpenStack.
```
Same issues exist in following

---

done